rust/config/testnet_config.json

```diff
@@ -68,6 +68,129 @@
       "timelockController": "0x0000000000000000000000000000000000000000",
       "validatorAnnounce": "0x3726EE36a2A9e11a40d1ffD7D9A1A16e0154cDA0"
     },
+    "arbitrumsepolia": {
+      "aggregationHook": "0xD2670EedcD21116c6F0B331Ce391eA4B3Bf1aB19",
+      "blockExplorers": [
+        {
+          "apiUrl": "https://api-sepolia.arbiscan.io/api",
+          "family": "etherscan",
+          "name": "Arbiscan",
+          "url": "https://sepolia.arbiscan.io"
+        }
+      ],
+      "blocks": {
+        "confirmations": 1,
+        "estimateBlockTime": 3,
+        "reorgPeriod": 0
+      },
+      "chainId": 421614,
+      "deployer": {
+        "name": "Abacus Works",
+        "url": "https://www.hyperlane.xyz"
+      },
+      "displayName": "Arbitrum Sepolia",
+      "domainId": 421614,
+      "domainRoutingIsm": "0xaCA2f65aFDa2cbC8BF28DdE096eCF83aCd121c0b",
+      "domainRoutingIsmFactory": "0xd785272D240B07719e417622cbd2cfA0E584d1bd",
+      "fallbackRoutingHook": "0xAb9B273366D794B7F80B4378bc8Aaca75C6178E2",
+      "index": {
+        "from": 49690504
+      },
+      "interchainGasPaymaster": "0xc756cFc1b7d0d4646589EDf10eD54b201237F5e8",
+      "interchainSecurityModule": "0x29cEAFEE1F76B9CE271750f86B2bD12C23F9dDb6",
+      "isTestnet": true,
+      "mailbox": "0x598facE78a4302f11E3de0bee1894Da0b2Cb71F8",
```

Comment:
This is the old mailbox address that kunal deployed a couple months ago right?

Curious if you ran into any issues or if you remember if the CLI was used by Kunal to do this originally

Reply:
Yup this is reusing kunal's deployment - we were able to pickup the existing artifacts as normal without issue!